Core/src/org/sleuthkit/autopsy/textextractors/TextFileExtractor.java

```diff
@@ -73,7 +76,22 @@ private Reader getReader(Charset encoding) {
     }
 
     @Override
-    public boolean isSupported() {
-        return file.getMIMEType().equals("text/plain");
+    public boolean isSupported() {        
+        // get the MIME type
+        String mimeType = file.getMIMEType();
+        
+        // if it is not present, attempt to use the FileTypeDetector to determine
+        if (StringUtils.isEmpty(mimeType)) {
+            FileTypeDetector fileTypeDetector = null;
+            try {
+                fileTypeDetector = new FileTypeDetector();
+            } catch (FileTypeDetector.FileTypeDetectorInitException ex) {
+                logger.log(Level.SEVERE, "Unable to create file type detector for determining MIME type.");
```

Comment:
We should pass the exception to the logger as the third argument so that a stack trace is logged.